### PR TITLE
Retire Common Crawl module & DAGs

### LINKED ISSUE #534: Remove non-essential buckets in development
## Problem

Currently we create a number of buckets on startup in Minio in dev (`commonsmapper`, `commonsmapper-v2`, `commoncrawl`, etc). This can be distracting for new devs.

## Description

We should see if these buckets can be dropped or if they're necessary at all currently (given that the commoncrawl isn't running).

## Additional context

https://github.com/WordPress/openverse-catalog/pull/516#pullrequestreview-979350718

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### LINKED ISSUE #846: Retire Common Crawl DAGs
## Description

We have two Common Crawl related DAGs available in production but not enabled: `commoncrawl_etl_workflow` and `sync_commoncrawl_workflow`. We currently don't have re-enabling them as part of our future priorities, so it might be best just to retire them for now to de-clutter the DAGs view. We'll still keep the code, but we would no longer need to navigate around them in the UI :slightly_smiling_face:

I believe we could move the entirety of the `dags/commoncrawl/` module into the retired folder to address this.

@WordPress/openverse-catalog - any concerns about doing this?

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #846 by @AetherUnbound 
Fixes #534 by @AetherUnbound 

## Description

Based on #861 from @Pmeet

This pull request has made the following changes
1. Retired the dags/commoncrawl module to retired folder
2. Removes the associated tests
3. Updates the README.md & env.template to remove any commoncrawl specific attributes
4. Removes unused testing buckets
5. Removes unused test files

## Testing Instructions

1. `just recreate` && `just test`
2. `just up` -> verity the commoncrawl DAGs no longer show up in the list.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Pmeet** [comment]
Thank you for doing this @AetherUnbound

**Comment by AetherUnbound** [comment]
> Thank you for doing this @AetherUnbound 

Thanks for contributing the original changes 😄

**Review by krysal — APPROVED** [review]
LGTM. Nice touch on updating the documentation to the past tense.

Thanks for your contribution as well @Pmeet!

File: README.md [hunk 1]
```diff
@@ -45,26 +58,13 @@ series of parquet files that contain:
 - the location of the webpage in the WARC file so that the page contents can be
   found.
 
-The steps above are performed in [`ExtractCCLinks.py`][ex_cc_links].
+The steps above are were in [`ExtractCCLinks.py`][ex_cc_links].
```
Comment (obulat) [inline README.md:61]:
```suggestion
The steps above were performed in [`ExtractCCLinks.py`][ex_cc_links].
```

**Review by obulat — APPROVED** [review]
Nice!

**Pull request merged**